<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CellarstoneTradingApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="pageBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAssets">
    <property name="title">
     <string>Assets</string>
    </property>
    <addaction name="actionStocksAsset"/>
    <addaction name="actionFuturesAsset"/>
   </widget>
   <widget class="QMenu" name="menuWatchlists">
    <property name="title">
     <string>Watchlists</string>
    </property>
    <addaction name="actionStocksWatchlist"/>
    <addaction name="actionFuturesWatchlist"/>
    <addaction name="actionOptionsWatchlist"/>
   </widget>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionHomePage"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionThreads"/>
    <addaction name="actionRealtimeData"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionManual_Calc"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuAssets"/>
   <addaction name="menuWatchlists"/>
   <addaction name="menuOptions"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStocksAsset">
   <property name="text">
    <string>Stocks</string>
   </property>
  </action>
  <action name="actionFuturesAsset">
   <property name="text">
    <string>Futures</string>
   </property>
  </action>
  <action name="actionStocksWatchlist">
   <property name="text">
    <string>Stocks</string>
   </property>
  </action>
  <action name="actionFuturesWatchlist">
   <property name="text">
    <string>Futures</string>
   </property>
  </action>
  <action name="actionOptionsWatchlist">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionHomePage">
   <property name="text">
    <string>HomePage</string>
   </property>
  </action>
  <action name="actionThreads">
   <property name="text">
    <string>Threads</string>
   </property>
  </action>
  <action name="actionManual_Calc">
   <property name="text">
    <string>Manual Calc</string>
   </property>
  </action>
  <action name="actionRealtimeData">
   <property name="text">
    <string>Realtime data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
